packages/frolint/src/utils/__tests__/prettier.test.ts

```diff
@@ -0,0 +1,44 @@
+import fs from "fs";
+import { resolve } from "path";
+
+import { applyPrettier } from "../prettier";
+
+const { writeFileSync, writeFile, unlinkSync } = fs;
+
+describe("prettier", () => {
+  /** Temporary file for running tests */
+  const testFilename = resolve(__dirname, "../testcontent.js");
+  /** Example content that prettier should be trying to fix */
+  const baseContent = `const test = {
+} ;`;
+
+  function createTestFile() {
+    (writeFileSync || writeFile)(testFilename, baseContent);
```

Comment:
Had to use different APIs for different Node.js version. `writeFileSync` is not available in v12